ui/widgets/autocomplete.js

```diff
@@ -539,9 +542,10 @@ $.widget( "ui.autocomplete", {
 		// Size and position menu
 		ul.show();
 		this._resizeMenu();
+		var pos = this.isRtl() ? { my: "right top", at: "right bottom", collision: "none" } : this.options.position;
```

Comment:
That would make the option unusable if you actually want it based on your design to be the same in either direction. We should never second guess users they are the ones that control both the direction an the option.